rest_framework/routers.py

```diff
@@ -290,6 +297,124 @@ def get_urls(self):
         return ret
 
 
+if path is not None:
```

Comment:
Since we're moving to 3.11, you can go ahead and remove the Django 1.11 compat checks.

Reply:
Yes, no problem.